lib/content-services/src/lib/i18n/ar.json

```diff
@@ -626,5 +626,20 @@
   "TAG_NODE_LIST": {
     "VIEW_MORE": "عرض {{ count }} المزيد"
   },
-  "swsdp": "عينة: مشروع تصميم موقع الويب"
+  "swsdp": "عينة: مشروع تصميم موقع الويب",
+  "ADF": {
+    "MESSAGES":{
```

Comment:
Can we maybe change the name to `ADF_LIBRARY_MEMBERSHIP_MESSAGES` as all of those strings are related to library membership directive? In this way we can also remove `ADF` level above

Reply:
yes, we should scope it somehow, either `ADF` or `ADF_LIBRARY_MEMBERSHIP_MESSAGES`, so that it does not clash with other features

Reply:
@MichalKinas @DenysVuika I've changed name to ADF_LIBRARY_MEMBERSHIP_MESSAGES, as there were other ADF_<something> names, however sonarcloud sees it as too many duplicated lines

Reply:
It looks false-positive to me, you didn't duplicated any code and it treats existing parts of code as duplications so we probably we can ignore it, @DenysVuika do you agree?

Reply:
@DenysVuika When you will have a few spare minutes, please have a quick look to verify if it can be merged. If yes, please do so, so I can move on to ACA counterpart of this PR.